experiments/ClimaEarth/components/land/climaland_bucket.jl

```diff
@@ -193,6 +198,9 @@ function Interfacer.get_field(bucket_sim::BucketSimulation, ::Val{:water})
        ρ_cloud_liq  # kg water / m2
 end
 
+function Interfacer.update_field!(sim::BucketSimulation, ::Val{:surface_temperature})
+    parent(sim.integrator.p.bucket.T_sfc) .= parent(field)
+end
```

Comment:
Is this still needed?

Reply:
No; this is no longer necessary since we use the `climaland_bucket` file to update this temp value.